WcaOnRails/spec/factories/competitions.rb

```diff
@@ -44,5 +44,10 @@
       with_delegate
       isConfirmed true
     end
+
+    trait :confirmed_and_visible do
```

Comment:
This is a little weird. There already is a `confirmed` trait, so a separate `visible` trait would make more sense to me. If you want both, you can pass in `[:confirmed, :visible]` to `FactoryGirl.create`, which is the same number of characters.